stack.yaml

```diff
@@ -43,6 +43,12 @@ extra-deps:
 - hookup-0.2.2
 - irc-core-2.5.0
 - discord-haskell-0.7.1
+# use time-1.9.2 from github
+- github: haskell/time
+  commit: 9e96c26132fef01a3113c8b152b1be96c0eccd86
```

Comment:
Why can't we just do `time-1.9.2` like everything above?

Reply:
time-1.9.2 doesn't exists in the stackage snapshot, so we must pull them other than stackage.

Reply:
Nevermind.  No need pull from github. 😅